src/server/pachyderm_test.go

```diff
@@ -12311,6 +12401,7 @@ func getUsablePachClient(t *testing.T) *client.APIClient {
 		if err == nil {
 			return client
 		}
+		time.Sleep(5 * time.Second)
```

Comment:
do we still need this?

Reply:
I think this is still useful - on my machine the reconnect would error immediately and it wouldn't wait long enough for the new pods to come up (even with trying 10x).

Reply:
Alright, would one of our Backoff functions be a better fit here then?

Reply:
Done